storage-access.bs

```diff
@@ -129,6 +129,14 @@ A {{Document}} is in a <dfn>first-party-site context</dfn> if it is the [=active
 
 A {{Document}} is in a <dfn>third party context</dfn> if it is not in a [=first-party-site context=].
 
+Let <dfn export abstract-op>whether the user agent explicitly allows unpartitioned cookie access</dfn> be an algorithm that, given a [=tuple=] |tuple| consisting of two [=sites=], runs the following steps. This algorithm returns "`none`", "`allow`" or "`disallow`".
```

Comment:
nit: I think it's more idiomatic to say something like 

```suggestion
To <dfn>determine whether the user agent explicitly allows unpartitioned cookie access</dfn>, given a [=tuple=] |tuple| consisting of two [=sites=], run the following steps. This algorithm returns "`none`", "`allow`" or "`disallow`".
```

(I also don't think the export attribute is needed here)